packages/server/controllers/events.ts

```diff
@@ -1,16 +1,17 @@
 // const postgres = require("../utils/postgres");
-const prisma = require("../utils/prisma");
+import prisma from "../utils/prisma";
+import { Event } from ".prisma/client/index";
```

Comment:
your imports for the `Event` type and `Guild` type in the `guilds.ts` file seem to be broken. They showed up as errors on my editor and the correct import was
```javascript
import { Event } from "@prisma/client";
import { Guild } from "@prisma/client";
```

These types are automatically generated by the Prisma client so make sure you have Prisma client generated by running `yarn workspace server prisma generate` in icebreak root directory.